<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings menu</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(114, 4, 85), stop: 1 rgb(3, 6, 55) )</string>
  </property>
  <widget class="QPushButton" name="color_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Open color picker</string>
   </property>
  </widget>
  <widget class="QPushButton" name="slider_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Open opacity slider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_btn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>Change Background</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
